app/models/ascent.rb

```diff
@@ -18,6 +18,7 @@ class Ascent < ApplicationRecord
   validates :ascent_status, inclusion: { in: AscentStatus::LIST }
 
   scope :made, -> { where.not(ascent_status: :project) }
+  scope :lead, -> { where(roping_status: [:lead_climb, :multi_pitch_leader, :multi_pitch_alternate_lead]) }
```

Comment:
histoire de compléter la méthode, on va aussi faire un scope `second` qui de la même manière récupérer les moulinettes et les seconds de cordé

Reply:
"Active Record already defines a method named `second`, which is used to fetch the second record from a query (similar to `first`)."  ==> je la nomme donc noLead